### PR TITLE
Add --classify command for command line tool

### PR DESCRIPTION
**Pull request by xulaus**

### CODE REVIEW

File: CHAID/__main__.py [hunk 1]
```diff
@@ -25,7 +26,16 @@ def main():
...
     if nspace.min_samples:
         config['min_sample'] = nspace.min_samples
-    CHAID.from_pandas_df(df, nspace.independent_variables, nspace.dependent_variable[0], **config).print_tree()
+    tree = CHAID.from_pandas_df(df, nspace.independent_variables, nspace.dependent_variable[0], **config)
+
+    if nspace.classify:
+        predictions = pd.Series(tree.predict())
+        predictions.name = 'node_id'
+        df = pd.concat([df, predictions], axis=1)
+        print(df.to_csv())
```
Comment (Rambatino) [inline CHAID/__main__.py]: Is there not a better way to print this?

Comment (xulaus) [inline CHAID/__main__.py]: Not to terminal. `to_csv` takes a file name argument if we want to go that route. The brackets are ugly but needed for python 3 compatibility

Comment (Rambatino) [inline CHAID/__main__.py]: @xulaus can we get this in if it's good?

Comment (xulaus) [inline CHAID/__main__.py]: Ill retest today, the interface has changed a bit I think.

**Comment by Rambatino** [comment]
👍

**Pull request merged**